insights/client/phase/v1.py

```diff
@@ -56,9 +56,16 @@ def get_phases():
 @phase
 def pre_update(client, config):
 
-    # Check if BASIC auth is used to print a WARNING message
-    if config.authmethod == 'BASIC':
-        logger.warning('WARN: BASIC authentication method is being deprecated. Please consider using CERT authentication method.')
+    # Check if the host is registered with subscription-manager
+    if config.authmethod == 'BASIC' and not _get_rhsm_identity():
+        logger.error('This host has not yet been registered, please ensure '
+                    'that the system is\nregistered with subscription-manager '
```

Comment:
That newline break in the middle of the sentence doesn't look good when your terminal is wider than 72 characters, it's fine without it.